src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4675,6 +4695,10 @@ int wm_vuldet_set_agents_info(agent_software **agents_software, update_node **up
             continue;
         }
 
+        if (!os_platform) {
+            mdebug1("Failed to get os_platform from agent");
```

Comment:
Change it to `mtdebug1(WM_VULNDETECTOR_LOGTAG, "Failed to get os_platform from agent");`

In Wazuh modules the tag should be added apart from the message.